web-client/src/views/AdvancedSearch/PractitionerSearchByName.tsx

```diff
@@ -66,7 +75,197 @@ export const PractitionerSearchByName = connect(
+                        {type}
+                      </label>
+                    </div>
+                  ))}
+                </div>
+              </fieldset>
+            </div>
+            <div className="grid-row grid-gap">
+              <fieldset className="usa-fieldset">
+                <legend className="usa-legend">
+                  Practice type <span className="usa-hint">(optional)</span>
+                </legend>
+                <div className="display-flex display-gap">
+                  {PRACTICE_TYPE_OPTIONS.map(practiceType => {
+                    return (
+                      <div className="usa-checkbox" key={practiceType}>
+                        <input
+                          checked={advancedSearchForm.practitionerSearchByName.practiceType?.includes(
+                            practiceType,
+                          )}
+                          className="usa-checkbox__input"
+                          id={`practiceType.${practiceType}`}
+                          name={`practiceType`}
+                          type="checkbox"
+                          value={practiceType}
+                          onChange={e => {
+                            updateAdvancedSearchFormValueSequence({
+                              formType: 'practitionerSearchByName',
+                              key: 'practiceType',
+                              value: e.target.value,
+                              isArray: true,
+                            });
+                          }}
+                        />
+                        <label
+                          className="usa-checkbox__label margin-top-0"
+                          htmlFor={`practiceType.${practiceType}`}
+                        >
+                          {practiceType}
+                        </label>
+                      </div>
+                    );
+                  })}
+                </div>
+              </fieldset>
+            </div>
 
+            <div className="grid-row grid-gap">
+              <div className="tablet:grid-col-6 margin-bottom-4">
+                <div className="margin-bottom-1">
+                  <label
+                    className="usa-label"
+                    htmlFor="admission-status"
+                    id="admission-status-label"
+                  >
+                    Admission Status{' '}
+                    <span className="usa-hint">(optional)</span>
+                  </label>
+                  <SelectSearch
+                    aria-labelledby="admission-status-label"
+                    data-testid="admission-status-filter"
+                    inputId="admission-status"
+                    name="admissionStatus"
+                    options={ADMISSIONS_STATUS_OPTIONS.map(admissionStatus => ({
+                      label: admissionStatus,
+                      value: admissionStatus,
+                    }))}
+                    placeholder="- Select one or more -"
```

Comment:
We use this in a few places now. I wonder if it is worth pulling out as a constant.